rest_local.yml

```diff
@@ -0,0 +1,28 @@
+---
+- hosts: all
+  become: false
+  vars_files:
+    - vars/main_local.yml
+    - "{{ rest_private_dir }}/conf/release_ebi.yml"
+
+  vars:
+    PERL_RC: "{{ ensembl_install_dir }}/activate_ensembl"
+    LANG: C
+    SHARED_PATH: "{{ ensembl_install_dir }}"
+    PROGRESSIVE_CACTUS_DIR: "{{ SHARED_PATH }}/paths/progressiveCactusFake/"
```

Comment:
What's the Fake part for?

Reply:
You're right, this should be removed. Let it fall through to PROGRESSIVE_CACTUS_DEFAULT in the actual role.

Reply:
Removed PROGRESSIVE_CACTUS_DIR.